app/service/data_svc.py

```diff
@@ -65,6 +65,27 @@ def __init__(self, dao):
                 for fact in source['facts']:
                     fact['source_id'] = source_id
                     await self.create_fact(**fact)
+                relationships = source.get('relationships')
```

Comment:
could these 4 lines be written as:
```
                for relationship in source.get('relationships', dict()).values():
                    await self.create_relationship(relationship)
```

Reply:
@privateducky GitHub introduced multi line comments fyi. hit the plus to comment like normal and then drag down.